modular_skyrat/modules/bsrpd/code/bsrpd.dm

```diff
@@ -15,6 +15,7 @@
 	var/current_capacity = BSRPD_CAPACITY_MAX
 	var/ranged_use_cost = BSRPD_CAPACITY_USE
 	var/in_use = FALSE
+	var/toggle = FALSE
```

Comment:
In addition to changing the var name, code doc this

Reply:
already did so, but my provider died yesterday so i couldnt push lol